recipe/meta.yaml

```diff
@@ -34,23 +36,12 @@ source:
     - patches/fix_map_anonymous.patch
     - patches/cmake-find-ideep-mkldnn-in-sourcetree.patch # [x86] #required for mkl implementation
     - patches/suppress_Wbitwise_instead_of_logical_clang.patch
-    # This is required in order to use our Pybind instead of the one
-    # submoduled into the Pytorch repo
-    - patches/cmake-fix-pybind-option.patch
-    # This patch was taken from conda-forge. There was a variable
-    # which directed the build system to use mkldnn for caffe2
-    # specifically, but since it was just turned on by the global
-    # mkldnn variable anyway, it was superfluous.
-    # Some of the changes in the original patch had been upstreamed.
-    - patches/cmake-unify-USE_MKLDNN-and-CAFFE2_USE_MKLDNN.patch
 
 build:
-  # Trigger 1
-  number: 1
-  # Automated s390x builds fail. The exact reason is unknown but probably due
-  # to resource constraints (the build is very memory intensive). Manual builds
-  # may or may not go through, your mileage may vary.
+  number: 0
   skip: True  # [py<37]
+  # ppc requires gcc v8 (see cbc.yaml). Python 3.11 is incompatible with anything built with gcc<11 because of runtime conflicts.
+  skip: True  # [(linux and ppc64le) and py==311]
   # TODO: note that at the moment we only use CUDA backend, e.g. would be different for mac
```

Comment:
Are there missing `entry_points`? See https://github.com/pytorch/pytorch/blob/49444c3e546bf240bed24a101e747422d1f8a0ee/setup.py#L944-L950

Reply:
thanks, added

Reply:
Would added entry points cause a build to fail? I'm just wondering if I need to run the CI on this PR again before merge, after this change.